test/cli.js

```diff
@@ -0,0 +1,31 @@
+
+'use strict'
+const { suite, test, run } = require('mocha')
+const assert = require('assert')
+const Loggerr = require('loggerr')
+const spy = require('through2-spy')
+const cli = require('../lib/cli')
+
+suite('cli validate', async () => {
+  test('check this support information', (done) => {
+    const logChecker = spy({ wantStrings: true }, function (chunk) {
+      try {
+        assert.ok(/information is valid/.test(chunk))
```

Comment:
I'm not a mocha expert, so if you know a better assertion lib feel free to suggest it 👍